packages/fxa-content-server/tests/functional.js

```diff
@@ -6,20 +6,15 @@ const testsSettings = require('./functional_settings');
 module.exports = testsSettings.concat([
   'tests/functional/fx_browser_relier.js',
   'tests/functional/oauth_webchannel.js',
-  'tests/functional/reset_password.js',
```

Comment:
It seems like most of these tests were ported to playwright. I am assuming the ones that weren't we decided to take down. That being said, are these deletions outside the scope of this PR? Are these tests rleated to syncV3 or signup? If this is just clean up, it might be good to note this in the PR description.